.github/workflows/release-please.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Release Please
+        uses: googleapis/release-please-action@v3
+        with:
+          token: ${{ secrets.KURTOSIS_BOT_TOKEN }}
```

Comment:
we will need to add the token for the Kurtosis Bot Token to our settings for this to work

Reply:
@lostbean